backend/src/main/java/com/bakdata/conquery/apiv1/QueryProcessor.java

```diff
@@ -416,4 +424,66 @@ public FullExecutionStatus getSingleEntityExport(Subject subject, UriBuilder uri
 	}
 
 
+	/**
+	 * Execute a basic query on a single concept and return only the included entities Id's.
+	 */
+	public Stream<Map<String, String>> resolveEntities(Subject subject, List<FilterValue<?>> filters, Dataset dataset) {
```

Comment:
Wir sollten echt überlegen ob wir nicht einen synchronen/long-poll execution modus anbieten.

Reply:
ja, wobei hier ja das format ja auch ein bestimmtes ist. Long-poll könnte man ja recht einfach über einen query-param in den query-submission Endpunkten machen, und dann mit der gleichen Logik wie hier.